### PR TITLE
Improvement to your "WebPage"

### PR DESCRIPTION
**Pull request by jhauswald**
When clicked, the cat meow meows. I provided a sample but I really do think you should opt for a better meow, specifically there are some good meows here: http://www.soundsnap.com/tags/meow.

Please let me know if you have any trouble or changes requested. Also this has only been tested locally, please test on your server.

### CODE REVIEW

**Comment by jhauswald** [comment]
@ypkang please help @yqzhang review.

File: index.html [hunk 1]
```diff
@@ -29,6 +29,13 @@
       <link rel="stylesheet" href="css/style.css" />
       <link rel="stylesheet" href="css/style-xlarge.css" />
     </noscript>
+    <script type="text/javascript">
```
Comment (yqzhang) [inline index.html]: Could you change these following lines to 2-space tabs?

File: index.html [hunk 2]
```diff
@@ -37,7 +44,9 @@
       <!-- Header -->
         <section id="header" class="skel-layers-fixed">
           <header>
-            <span class="image avatar"><a href="#"><img src="images/photo.jpg" alt="" /></a></span>
+          <span class="image avatar">
```
Comment (yqzhang) [inline index.html]: Could you reformat these lines in terms of indentation?

**Comment by yqzhang** [comment]
Is it possible to use [this meow](http://www.soundsnap.com/node/107823) instead? Thank you!

**Comment by jhauswald** [comment]
What about [this one](http://www.orangefreesounds.com/cat-sound/)? You need to pay for those other ones?

**Comment by yqzhang** [comment]
I've been trying to hack the website to get the audio file. I think I am pretty close and will keep you posted.

**Comment by jhauswald** [comment]
So... when is this getting merged in?

**Comment by yqzhang** [comment]
Now?

**Pull request merged**